event.go

```diff
@@ -0,0 +1,28 @@
+package xendit
+
+type EventInvoicePaid struct {
```

Comment:
Could this be renamed to InvoiceCallback to reflect what we define on the API Reference [here](https://developers.xendit.co/api-reference/#invoice-callback)? Thanks!

Could we also move this to the `invoice` folder over [here](https://github.com/xendit/xendit-go/tree/master/invoice) since this logic is specific to Invoices?

Reply:
sounds good ! 
i renamed it to `InvoiceCallback` and move it to invoice package.